spec/example_app/app/dashboards/host_dashboard.rb

```diff
@@ -0,0 +1,30 @@
+require "administrate/base_dashboard"
+
+class HostDashboard < Administrate::BaseDashboard
+  ATTRIBUTE_TYPES = {
+    id: Field::Number,
+    name: Field::String,
+    created_at: Field::DateTime,
+    updated_at: Field::DateTime,
```

Comment:
Just this lint issue and I'll merge this PR :pray:

Reply:
@pablobm thank you, I've fixed this lint issue